main.tf

```diff
@@ -370,6 +370,32 @@ resource "aws_iam_role" "instance" {
   tags = merge(local.tags, var.role_tags)
 }
 
+################################################################################
+### Policy for the instance to use the KMS key
+################################################################################
+resource "aws_iam_policy" "instance_kms_policy" {
+  count = var.enable_kms ? 1 : 0
```

Comment:
Introduction of the count let the policy created on line 472 fails when "enable_kms" is set

```
│ Error: Invalid count argument
│
│   on ../../main.tf line 472, in resource "aws_iam_role_policy_attachment" "docker_machine_cache_instance":
│  472:   count = var.runners_executor == "docker+machine" ? (var.cache_bucket["create"] || lookup(var.cache_bucket, "policy", "") != "" ? 1 : 0) : 0
│
│ The "count" value depends on resource attributes that cannot be determined until apply, so Terraform cannot predict how many instances will be created. To
│ work around this, use the -target argument to first apply only the resources that the count depends on.
╵
╷
```

Reply:
Didn't see this behavior here. Switched from 6.2.0 to this version without any problems. Have `enable_kms` set and tried with/without `kms_key_id`.

Reply:
@npalm How to proceed here? Any chance to reproduce it? To me it seems not related to this change.

Haven't we removed these `lookup`s in the past? May be we forgot this one.

Reply:
My bad, ran everything again without a problem